views/common/ratingChooser.ejs

```diff
@@ -1,28 +1,33 @@
-<form>
-    <div class="form-check row">
-        <% for (var rating in ratings) { %>
-            <span class="col-sm-2">
-                <input class="form-check-input" type="radio" name="securityRating" id=<%="rating" + rating %> value=
-                <%=rating%> >
-                    <label class="form-check-label" for=<%="rating" + rating %>>
-                        <%= "Level " + rating %>
-                    </label>
-            </span>
-            <% } %>
+<% if (securityRating == null) { %>
+    <div class='markdown'>
+        Note:  
+        As of this date, we have not expanded on "<%=vuln_title%>" section.
+        As such, the Security Rating toggle feature is not available for this section (i.e., page).  
+        You may treat this section as if it were at Security Rating 0 (i.e., vulnerable). 
+        This section is exactly the same as found in the original Appsecco DVNA.
     </div>
-    <div>
-        <button type="submit">Submit</button>
```

Comment:
Good point.  I think I explained the 'Sample Rest Link for "user" at Security Level 0' in my gitbook, but I'll add some info to make that clearer.

Reply:
Thanks for checking. -- Yes, I intended to delete the submit button.  

Personally, the radio options and submit button was too clunky for me.  Sometimes, I would click on the radio option (e.g., security rating 1), and it would show that I chose the appropriate option (i.e., security rating 1), but it would stay at the wrong security rating (e.g., security rating 0) because I forgot to press the submit button ... ending up with functionality I did not intend.

My hope was to get rid of one extra step, and also to notify user better of what security rating was chosen.  Here, pressing a button will both submit and have UX feed back (green indicating that it was chosen, and orange text confirming).

However, I realize that this was based solely on my personal UX, so if you don't like it, I'm happy to change it back to the radio buttons.

Note:  I kept your iterating option, so if you add additional security ratings (e.g., security rating 2), it would automatically update.  That was good coding your part, and I decided to keep it.

Hope that clarifies things.